### PR TITLE
Centralize Documentations and README.md

### PR DESCRIPTION
**Pull request by SomtochiAma**
This pull request is for centralizing the documentation in cluster-addons and making the documentation more interesting and clearer to whoever is poking around. Reviews and suggestions are welcome from everyone @rajansandeep @dholbach @stealthybox @justinsb .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ This sub-project wants to figure out the best way to install, manage and deliver
 
 In this repository we explore ideas for all of the above. [Cluster addon operators](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cluster-lifecycle/addons/0035-20190128-addons-via-operators.md) in particular.
 
+With cluster addon operators, we are exploring a kubernetes-native way of managing addons using CRDs(Custom Resource Definitions) and controllers where the controllers encode how best to manage the addon. Installing and managing an addon could be as simple as creating a custom resource.
+
+We are also exploring other concept such as managing different simple addons with a single controller to reduce consumption of resources and make it dead simple to manage addons. We welcome you to create your own addon operators and try using the ones created by this community.
```
Comment (dholbach) [inline README.md:11]:
```suggestion
We are also exploring other concepts such as managing different simple addons with a single controller to reduce consumption of resources and make it dead simple to manage addons. We welcome you to create your own addon operators and try using the ones created by this community.
```

File: dashboard/README.md [hunk 2]
```diff
@@ -1,65 +1,35 @@
...
+- 2.0.0
 ```
 
-## 3. Delete unnecessary test files
-Delete the test suites that are checking that kubebuilder is working:
+Stable version - `2.0.0`
 
-```bash
-find . -name "*_test.go" -delete
+## Usuage
```
Comment (rajansandeep) [inline dashboard/README.md:15]:
```suggestion
## Usage
```

**Comment by justinsb** [comment]
I like this approach - thanks @SomtochiAma 

/approve
/lgtm

**Pull request merged**